graphstorm-processing/graphstorm_processing/repartition_files.py

```diff
@@ -825,27 +835,16 @@ def modify_flat_array_metadata(
     """
     edge_data_meta: dict = metadata_dict.get("edge_data", {})
     node_data_meta: dict = metadata_dict.get("node_data", {})
-
-    def key_is_mask(key: str):
-        for mask in [
-            "train_mask",
-            "val_mask",
-            "test_mask",
-        ]:
-            if key.startswith(mask):
-                return True
-        return False
+    graph_info: dict = metadata_dict["graph_info"]
```

Comment:
Is metadata_dict per edge type? or per node type?

Reply:
Do you have a docstring describing the meaning of each field of graph_info?

Reply:
Part of metadata_dict is structured as the "chunked graph format" of DGL: https://docs.dgl.ai/guide/distributed-preprocessing.html#chunked-graph-format

We add the additional key `graph_info` where we hold a different representation of the graph information in a more readable/verbose format. I'll add some documentation on its contents.